interface/patient_file/summary/add_edit_amendments.php

```diff
@@ -116,9 +116,6 @@
 <?php Header::setupHeader('datetime-picker'); ?>
 
 <style>
-.highlight {
-  color: var(--success);
-}
 tr.selected {
   background-color: var(--white);
 }
```

Comment:
Hi @tywrenn 

I guess this CSS style can be replaced with `table-hover`, What do you think?

Reply:
Yep that can be replaced with `table-hover`